### ISSUE TITLE
Major contrast bug (Chromium v88)

### ISSUE THREAD

**Issue by niftylettuce** [GoogleChrome/lighthouse#12091]
As you can see from these screenshots, a high contrast button with white text on a black background (and even enlarged with bolder font face) causes Accessibility score to be 98 instead of 100.
<img width="943" alt="Screen Shot 2021-02-14 at 2 05 16 AM" src="https://user-images.githubusercontent.com/523141/107871693-8e1fc400-6e69-11eb-983a-69566322ff29.png">
<img width="862" alt="Screen Shot 2021-02-14 at 2 05 04 AM" src="https://user-images.githubusercontent.com/523141/107871694-8eb85a80-6e69-11eb-9838-1139fe314dde.png">
<img width="818" alt="Screen Shot 2021-02-14 at 1 51 45 AM" src="https://user-images.githubusercontent.com/523141/107871695-8eb85a80-6e69-11eb-913d-58b074954eee.png">

https://forwardemail.net
<img width="964" alt="Screen Shot 2021-02-14 at 2 07 15 AM" src="https://user-images.githubusercontent.com/523141/107871691-8cee9700-6e69-11eb-9333-0dbed271d81b.png">

**Comment by niftylettuce** [comment]
It was referencing the wrong element on click for some reason.
